app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: :user_id
```

Comment:
- [ ]  Previous reviewer suggested you to use AuthorId instead of userId, and i just think normiculture is not a big deal thing, but what important is to consider how he suggested you to create a relationship, it is a very clean and good way of creating relationship between comment and user table . Use the suggested style to match with the provided ERD diagram 
![image](https://github.com/Muhammad0602/Blog-app/assets/43877689/9792fb6f-19d8-469f-aaf9-df2d472b08a8)

Reply:
I know this way connecting them and as you can see I used it for connecting posts and user. But can you please check the ERD I provided on the PR. It is userId not authorId.

Reply:
@ishimwezachee,
What should I do now?

Reply:
@Muhammad0602  I think @ishimwezachee didn't see my comments above. He could have checked your ERD. You'll need to kindly submit once more and sorry for the inconvenience.

Reply:
@FrederickMih But I already did the other changes and attached the ERD to PR before this request. And I have only one more request left

Reply:
and I really appreciate the feedback, but I was confused instead of learning.